templates/index.tpl

```diff
@@ -0,0 +1,56 @@
+{**
+ * plugins/generic/backup/templates/index.tpl
+ *
+ * Copyright (c) 2014-2018 Simon Fraser University
+ * Copyright (c) 2000-2018 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * List of operations this plugin can perform
+ *}
+
+<script src="{$pluginJavaScriptURL}/BackupDownloadHandler.js"></script>
```

Comment:
I'd suggest using `TemplateManager::addJavascript` rather than manually including this here.

Reply:
(see note below on the PKP JS framework before doing this)